glean-core/ffi/examples/glean_app.c

```diff
@@ -31,9 +31,23 @@ int main(void)
 
   glean_counter_add(metric, 2);
 
-  char *payload = glean_ping_collect(store1, NULL);
-  printf("Payload:\n%s\n", payload);
-  glean_str_free(payload);
+  glean_submit_ping_by_name("store1", NULL);
+
+  // task == 0 (wait)
+  // task == 1 (upload)
+  // task == 2 (done)
+  FfiPingUploadTask task = glean_get_upload_task();
+  while (task.tag == 1) {
+    printf("tag: %d\n", task.tag);
+    if (task.tag == 1) {
+      printf("uuid: %s\n", task.upload.uuid);
```

Comment:
nit: let's collapse the uuid and path in a single line, would you? :-)

Reply:
Hm, I think that will make the output harder to visualize. The uuid is already in the path, in this case I can just remove it altogether and leave path and body only.

Reply:
Yeah, path and body sounds like enough.